core/src/test/java/kafka/admin/DeleteTopicTest.java

```diff
@@ -239,7 +239,7 @@ public void testDeleteNonExistingTopic(ClusterInstance cluster) throws Exception
     @ClusterTest(serverProperties = {
         @ClusterConfigProperty(key = "log.cleaner.enable", value = "true"),
         @ClusterConfigProperty(key = "log.cleanup.policy", value = "compact"),
-        @ClusterConfigProperty(key = "log.segment.bytes", value = "100"),
+        @ClusterConfigProperty(key = "log.segment.bytes", value = "1048576"),
```

Comment:
does this test really need this config? From what it looks like it does the following:
1. create a topic
2. Write 100 X 3 entries where 100 entries are duplicated
3. Waits for cleaning to complete until 0 offset <-- this looks wrong
4. Deletes the topic
5. Verifies topic deletion

Why do we need a segment roll here? (unless I am missing something)